### PR TITLE
Add release pipeline

### PR DESCRIPTION
**Pull request by movsic**

### CODE REVIEW

File: .github/scripts/get_min_versions.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import sys
+
+import tomllib
+from packaging.version import parse as parse_version
+import re
+
+MIN_VERSION_LIBS = ["langchain-core", "langchain-community", "langchain"]
```
Comment (lkuligin) [inline .github/scripts/get_min_versions.py:7]: do we need this?

File: .github/workflows/_release.yml [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+        run: make tests
+        working-directory: ${{ inputs.working-directory }}
+
+      - name: 'Authenticate to Google Cloud'
+        id: 'auth'
+        uses: google-github-actions/auth@v2
+        with:
+          credentials_json: '${{ secrets.GOOGLE_CREDENTIALS }}'
+
+      - name: Run integration tests
```
Comment (lkuligin) [inline .github/workflows/_release.yml:166]: let's disable this maybe until we don't migrate to new integration testing component?

File: .github/scripts/get_min_versions.py (hunk 1 above)
Comment (efriis) [reply .github/scripts/get_min_versions.py:7]: Good to test for core

File: .github/workflows/_release.yml (hunk 2 above)
Comment (efriis) [reply .github/workflows/_release.yml:166]: Would prefer to keep integration tests running before releases! What is the new component? New script?

**Review by efriis — APPROVED** [review]

**Pull request merged**